ansible/roles/test/files/ptftests/arista.py

```diff
@@ -520,3 +526,21 @@ def check_change_time(self, output, entity, what):
         # Note: the first item is a placeholder
         return 0, change_count
 
+    def get_bp_ip(self, v4=True):
```

Comment:
>bp [](start = 12, length = 2)

What is `bp`?

Reply:
backplane

Reply:
no longer using this method. nexthop ip is derived from the topo